modules/space/translation/horizonstranslation.cpp

```diff
@@ -186,4 +219,77 @@ void HorizonsTranslation::readHorizonsTextFile(const std::string& horizonsTextFi
     fileStream.close();
 }
 
+bool HorizonsTranslation::loadCachedFile(const std::string& file) {
+    std::ifstream fileStream(file, std::ifstream::binary);
+
+    if (!fileStream.good()) {
+        LERROR(fmt::format("Error opening file '{}' for loading cache file", file));
+        return false;
+    }
+
+    // Read how many values to read
+    int32_t nKeyframes = 0;
```

Comment:
I guess this would be `nValues` instead of `nKeyframes`?

Reply:
I actually mean "read how many keyframes to read" so i adjusted the comment instead